src/TheNextWeek/bvh.h

```diff
@@ -13,12 +13,20 @@
 
 #include "common/rtweekend.h"
 #include "hittable.h"
+#include <algorithm>
 
 
 class bvh_node : public hittable  {
     public:
-        bvh_node() {}
-        bvh_node(hittable **l, int n, double time0, double time1);
+        bvh_node() = delete;
```

Comment:
That's going to be a Yikes from me.

I think that deleting the default constructor is a good idea.
I just hate the committee's choice of syntax.
I think it's best to gloss over this detail, and lead it to the user to grokk what's happening

Reply:
Ehhh, I guess the syntax is fine.

I guess the `= delete;` activated my "this is outside our usual c++ calls" lizard brain.

Reply:
I was being pedantic about disallowing the construction of unspecified BVH nodes, but I can definitely see how I'm just adding a cognitive speed bump in this context. Would totally do this in production code, but will simplify.

Reply:
I'm just going to pull the parameterless ctor altogether, and ignore the fact that C++ will autogenerate the default empty constructor.